components/org.wso2.carbon.consent.mgt.core/src/test/resources/conf/consent-mgt-config.xml

```diff
@@ -21,19 +21,19 @@
         <Name>jdbc/WSO2CarbonDB</Name>
     </DataSource>
     <PIIController>
-        <PiiController>sample</PiiController>
-        <Contact>sample</Contact>
-        <Email>sample@abc.com</Email>
-        <Phone>+01433444333</Phone>
+        <PiiController>change-me</PiiController>
+        <Contact>change-me</Contact>
+        <Email>change-me</Email>
```

Comment:
Lets have a valid email 😉

Reply:
i mean valid dummy value

Reply:
@mefarazath: Can you please propose suitable values?